packages/@aws-cdk/custom-resources/test/aws-custom-resource/aws-custom-resource.test.ts

```diff
@@ -521,3 +521,47 @@ test('can specify log retention', () => {
     RetentionInDays: 7,
   });
 });
+
+test('can specify assumed role', () => {
```

Comment:
Can you also add a test for the runtime in `aws-custom-resource-provider.test.ts`?